commit 5de3781339f493ef02c9e70819baddbcc4c4ee9c (refs/changes/28/4628/2)
Author: Nick Zivkovic <nick.zivkovic@joyent.com>
Date:   2018-08-02T23:34:59+00:00 (1 year, 2 months ago)
    
    TRITON-661 CloudAPI needlessly imports util package
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/main.js b/main.js
index 51efc1a..e9650f8 100644
--- a/main.js
+++ b/main.js
@@ -199,4 +199,3 @@ if (PARSED.help) {
 
 // There we go!:
 run();
-var util = require('util');
